sort.c

```diff
@@ -10,30 +10,52 @@
 
 #include "redisraft.h"
 
-static void replySortedArray(RedisModuleCtx *ctx, RedisModuleCallReply * reply)
+static void replySortedArray(RedisModuleCtx *ctx, RedisModuleCallReply * reply, int reply_type)
```

Comment:
Missing function documentation.
Also: Why pass reply_type explicitly, when it can be inferred from the RedisModuleCallReply?

Reply:
ok re documenation

good Q. I was going to say I didn't know it could be inferred, but that's obviously wrong, as I infer it to get that value.  so only reason is simply to not infer it again, but probably not a good enough reason.